python-packages/smithy-core/tests/unit/test_documents.py

```diff
@@ -540,12 +557,14 @@ def test_is_none():
         "documentMember": {"target": DOCUMENT, "index": 7},
         "listMember": {"target": STRING_LIST_SCHEMA, "index": 8},
         "mapMember": {"target": STRING_MAP_SCHEMA, "index": 9},
+        "sparseListMember": {"target": SPARSE_STRING_LIST_SCHEMA, "index": 11},
+        "sparseMapMember": {"target": SPARSE_STRING_MAP_SCHEMA, "index": 12},
     },
 )
 SCHEMA.members["structMember"] = Schema.member(
     id=SCHEMA.id.with_member("structMember"),
     target=SCHEMA,
-    index=len(SCHEMA.members),
+    index=10,
```

Comment:
Is it worth adding a quick comment on line 560 above noting index 10 is added below?